File: mods/Simulated-Project/simulated/common/src/main/java/dev/simulated_team/simulated/content/blocks/void_anchor/VoidAnchorBlockEntity.java
package dev.simulated_team.simulated.content.blocks.void_anchor;

import net.minecraft.core.BlockPos;
import net.minecraft.world.level.block.entity.BlockEntity;
import net.minecraft.world.level.block.entity.BlockEntityType;
import net.minecraft.world.level.block.state.BlockState;

public class VoidAnchorBlockEntity extends BlockEntity {
    public VoidAnchorBlockEntity(final BlockEntityType<?> blockEntityType, final BlockPos blockPos, final BlockState blockState) {
        super(blockEntityType, blockPos, blockState);
    }
}

File: src/main/kotlin/dev/codinglabs/modpack/VoidAnchorBlockEntity.kt
package dev.codinglabs.modpack

import net.minecraft.core.BlockPos
import net.minecraft.nbt.CompoundTag
import net.minecraft.world.level.block.entity.BlockEntity
import net.minecraft.world.level.block.state.BlockState

class VoidAnchorBlockEntity(pos: BlockPos, state: BlockState) :
    BlockEntity(BlockEntities.VOID_ANCHOR, pos, state)
